releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -1,16 +1,13 @@
 # Microsoft.FeatureManagement
-[Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
-# Microsoft.FeatureManagement.AspNetCore
-[Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
+[Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
 ## 4.0.0-preview2 - March 7, 2024
 
-### Enchancements
+### Enhancements
 
-* Use flags to enable different service implementations in dependency injection. ([#39](https://github.com/microsoft/FeatureManagement-Dotnet/issues/39)). See more details [here](https://github.com/microsoft/FeatureManagement-Dotnet/tree/preview?tab=readme-ov-file#variants-in-dependency-injection).
-* Additional mechanisms to track targeting within telemetry to improve the connection between published events and metrics. ([#409](https://github.com/microsoft/FeatureManagement-Dotnet/issues/409))
-* Added support for Net8 applications by adding Net8 as a build target. ([#364](https://github.com/microsoft/FeatureManagement-Dotnet/issues/364))
+* Added support for variant feature flag-based service provider in dependency injection. It allows different service implementations to be injected automatically for different targeted audiences based on their variant assignment. ([#39](https://github.com/microsoft/FeatureManagement-Dotnet/issues/39)). See more details [here](https://github.com/microsoft/FeatureManagement-Dotnet/tree/preview?tab=readme-ov-file#variants-in-dependency-injection).
+* Added `TargetingContext` to evaluation events. This allows events to accuratly represent what the targeting context was at the time of feature evaluation. ([#409](https://github.com/microsoft/FeatureManagement-Dotnet/issues/409))
```

Comment:
Small tweak and correct the typo.

Added a `TargetingContext` property to the `EvaluationEvent`.  This allows feature evaluation events to accurately represent what the targeting context was at the time of feature evaluation